app/controllers/application_controller.rb

```diff
@@ -480,6 +480,13 @@ def wrapping_layout
     "application"
   end
 
+  def set_cache_headers
+    # prevents browser caching on pages set with before action
+    response.headers["Cache-Control"] = "no-cache, no-store"
+    response.headers["Pragma"] = "no-cache"
+    response.headers["Expires"] = "Mon, 01 Jan 1990 00:00:00 GMT"
+  end
```

Comment:
Not important, might be nice: I feel like there could be a clearer name for this method, like `set_no_cache_headers` `no_cache_headers` or `prevent_response_caching` or `do_not_cache_response`.